pipelines/toast_ground_sim.py

```diff
@@ -508,28 +135,16 @@ def parse_arguments(comm):
         '"fknee" (float, Hz), "alpha" (float), and '
         '"NET" (float).',
     )
-    parser.add_argument(
-        "--focalplane_radius",
-        required=False,
-        type=np.float,
-        help="Override focal plane radius [deg]",
-    )
     parser.add_argument(
         "--freq",
         required=True,
-        help="Comma-separated list of frequencies with " "identical focal planes",
-    )
-    parser.add_argument(
-        "--tidas", required=False, default=None, help="Output TIDAS export path"
-    )
-    parser.add_argument(
-        "--spt3g", required=False, default=None, help="Output SPT3G export path"
+        help="Comma-separated list of frequencies with identical focal planes",
     )
 
     try:
         args = parser.parse_args()
     except SystemExit:
-        return
+        sys.exit(0)
```

Comment:
should we exit with 1? or what about just leaving the exception be raised to the top?

Reply:
The main() has special exception handlers that I don't want to trigger in this case. The argument parser already prints out all of the useful information.